### PR TITLE
Update auth.py

### PR DESCRIPTION
**Pull request by pythexcel**

### CODE REVIEW

File: app/api/auth.py [hunk 1]
```diff
@@ -92,6 +77,16 @@ def login():
            result = response_user_details.json()
            user_data = result['data']['user_profile_detail']
            role_response = jwt.decode(token['data']['token'], None, False)
+           if role_response["role"] == "Admin":
+               URL = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
+               payload_all_user_details = {"action": "get_enable_user", "token": token['data']['token']}
+               response_all_user_details = requests.post(url=URL, json=payload_all_user_details)
+               result = response_all_user_details.json()
+               user = mongo.db.all_users.insert({
+                   "All_users": result
```
Comment (reactexcel) [inline app/api/auth.py]: no no. not like this. properly in users table only. don't take new collection

Comment (reactexcel) [reply app/api/auth.py]: all members works with users collection not some new collection

File: app/api/auth.py [hunk 2]
```diff
@@ -92,6 +77,16 @@ def login():
            result = response_user_details.json()
            user_data = result['data']['user_profile_detail']
            role_response = jwt.decode(token['data']['token'], None, False)
+           if role_response["role"] == "Admin":
+               URL = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
```
Comment (reactexcel) [inline app/api/auth.py]: move this URL to a global config file

File: app/config.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+URL = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
```
Comment (reactexcel) [inline app/config.py]: k. this is wrong.

there will be just based url. rest specific to api put url where code is called.
only one url will be here

File: app/api/auth.py [hunk 4]
```diff
@@ -56,7 +55,7 @@ def login():
       })
    if hr is not None and "integrate_with_hr" in hr:
 
-       URL_login = 'https://hr.excellencetechnologies.in/attendance/API_HR/api.php'
+       URL_login = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
```
Comment (reactexcel) [inline app/api/auth.py]: this should come from config

File: app/api/auth.py [hunk 5]
```diff
@@ -67,31 +66,27 @@ def login():
...
-       else:
-           pass
-        
-        
+         
        if token['data'] == {'message': 'Invalid Login'}:
            return jsonify(msg='invalid login')
        else:
-           URL_details = 'https://hr.excellencetechnologies.in/attendance/sal_info/api.php'
+           URL_details = 'http://dev.hr.excellencetechnologies.in/hr/attendance/sal_info/api.php'
```
Comment (reactexcel) [inline app/api/auth.py]: base url should come from config

File: app/api/auth.py (hunk 4 above)
Comment (pythexcel) [reply app/api/auth.py]: ok this one we can access from config.py

File: app/api/auth.py (hunk 5 above)
Comment (pythexcel) [reply app/api/auth.py]: but if we add this url in config.py then there will be two urls

**Pull request merged**